applications/openshift/registry/ocp_allowed_registries_for_import/oval/shared.xml

```diff
@@ -0,0 +1,43 @@
+{{% set YAML_TEST_OVAL_VERSION = [5, 11] %}}
+
+{{% if target_oval_version >= YAML_TEST_OVAL_VERSION %}}
+<def-group>
+	<definition class="compliance" version="1" id="{{{ rule_id }}}">
+      <metadata>
+        <title></title>
+        {{{- oval_affected(products) }}}
+        <description>Test allowed registries to import are specified</description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="Make sure that allowed registries for import are specified" negate="true" test_ref="test_{{{ rule_id }}}"/>
+        <criterion comment="Make sure that there is the actual file to scan" test_ref="test_file_for_{{{ rule_id }}}"/>
+      </criteria>
+    </definition>
+
+    <ind:yamlfilecontent_test id="test_{{{ rule_id }}}" check="at least one" comment="Find at least one match of allowedRegistriesForImport in cluster image settings" version="1">
+	    <ind:object object_ref="object_{{{ rule_id }}}"/>
+    </ind:yamlfilecontent_test>
+
+    <local_variable id="{{{ rule_id }}}_dump_location" datatype="string" comment="The actual filepath of the file to scan." version="1">
+       <concat>
+	       <variable_component var_ref="ocp_data_root"/>
+	       <literal_component>/apis/config.openshift.io/v1/images/cluster</literal_component>
+       </concat>
+    </local_variable>
+
+    <unix:file_test id="test_file_for_{{{ rule_id }}}" check="only one" comment="Find the actual file to be scanned." version="1">
+	    <unix:object object_ref="object_file_for_{{{ rule_id }}}"/>
+    </unix:file_test>
+
+    <unix:file_object id="object_file_for_{{{ rule_id }}}" version="1">
+      <unix:filepath var_ref="{{{ rule_id }}}_dump_location"/>
+    </unix:file_object>
+
+    <ind:yamlfilecontent_object id="object_{{{ rule_id }}}" version="1">
+      <ind:filepath var_ref="{{{ rule_id }}}_dump_location"/>
+      <ind:yamlpath>.spec.allowedRegistriesForImport[:]</ind:yamlpath>
```

Comment:
@matejak is this sufficient to test that at least one `allowedRegistriesForImport` exists?

Reply:
@evgenyz does this look sufficient to test that at least one `allowedRegistriesForImport` entry exists? Or is there another test required?

Reply:
Yes, that's correct